HerPublicWebsite.BusinessLogic/Models/ReferralRequest.cs

```diff
@@ -28,6 +28,7 @@ public class ReferralRequest
     public string ContactEmailAddress { get; set; }
     public string ContactTelephone { get; set; }
 
+    public bool PendingLocalAuthorityWhenSubmitted { get; set; }
```

Comment:
This name makes it sound like the field might return a `PendingLocalAuthority` rather than a boolean.

In this case, I think we want `WasSubmittedToPendingLocalAuthority`.

It's not a hard and fast rule, but booleans typically contain a verb like `Is…`, `Has…` or `Was…`.